### DISCUSSION TITLE
希望能加入自定义热键/组合键的功能，非常感谢

### DISCUSSION THREAD

**Discussion by charles7580** [taojy123/KeymouseGo#166] [Ideas]
希望能加入自定义热键/组合键的功能，非常感谢。有时候需要频繁启用脚本，键盘最上方的一排有点远。要是能自定义全局组合键就方便了（类似Alt+Q之类）。
另外当前版本按启用脚本的热键貌似无效，鼠标点击按钮有效。

**Comment by ZutJoe** [comment]
功能性的建议提issue吧，方便管理一点，谢谢

**Comment by Silverliu2023** [comment]
尊敬的开发者,

您好, 您编写的KeymouseGo,软件非常好用!

在使用过程中,希望能增加自定义Hotkey 热键, 比如 Alt+Q   , Ctrl + K, 等等可以自定义的热键。
鼠标有很多种很多款式，希望能够添加罗技G300S。

同花顺软件， F1 F2 F3 F4 F5 F6 F7 F8 F9 F10 F11 F12 都已经有定义了，容易和日常使用的软件冲突。

谢谢您的智慧和用户意见的采纳。

软件做得非常通俗易懂容易上手！

再次谢谢您的努力付出！

A股大牛市已来，务必请抓住！

KeymouseGo用户！
